lab07-jacob/lib/parse-body.js

```diff
@@ -0,0 +1,16 @@
+function parseBody(req, callback) {
+  req.body = '';
+  req.on('data', function(data) {
+    req.body += data.toString();
+  });
+  req.on('end', function() {
+    try {
+      req.body = JSON.parse(req.body);
+      callback(null, req.body);
+    } catch (err) {
+      callback(err);
+    }
+  });
+}
+
+module.exports = parseBody;
```

Comment:
Just assign the function directly to `module.exports` since this is the only piece of export code in the module.